docs/organizer-guide/meetups/starting.rst

```diff
@@ -1,11 +1,10 @@
 Starting a Meetup
 =================
 
-Interested in fostering a lovely gathering of documentarians in your neck of the woods?
-This is the place to start!
+If you've been to a Write The Docs conference, you may have the need to "continue the conversation".
+With a little help, you can make it happen with a Meetup in your area!
```

Comment:
Write the Docs [meetups](http://www.writethedocs.org/meetups/) provide a great way to connect with documentarians that you've met through our [conferences](http://www.writethedocs.org/conf/), [Slack channel](http://slack.writethedocs.org), and [forum](http://forum.writethedocs.org).  If you don't have a Write the Docs meetup nearby, we can help you get one started.